services/models/service.go

```diff
@@ -6,8 +6,18 @@ import (
 )
 
 type ServiceOverview struct {
-	Name         string `json:"name"`
-	IstioSidecar bool   `json:"istioSidecar"`
+	// Name of the workload
```

Comment:
"Name of the workload" at ServiceOverview level ?
It is "service" on this context, isn't it ?

Reply:
Copy paste fail ^^ it's for swagger